<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget#Form
{
	background-color: rgb(249, 249, 249);
}


QWidget
{
	padding: 2px;
}


QPushButton 
{
	background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 rgba(200, 200, 200, 255), 
stop:0.5 rgba(221, 221, 221, 255),
stop:1 rgba(200, 200, 200, 255));
	border-style: outset;
    border-width: 2px;
    border-color: rgba(234, 234, 234, 255)
 }

QPushButton#pushButton_02_buyitnow
{
	background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 rgba(234, 0, 0, 255), 
stop:0.5 rgba(255, 39, 39, 255),
stop:1 rgba(234, 0, 0, 255));
	color: rgb(249, 249, 249);
}

 QPushButton:pressed {
	background-color: rgb(221, 221, 221);
 }

 QPushButton:hover {
	 background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 rgba(229, 229, 229, 255), 
stop:0.5 rgba(221, 221, 221, 255),
stop:1 rgba(229, 229, 229, 221));
    border-color: rgba(221, 128, 32, 255)
 }


QLineEdit
{
	background-color:  rgba(230, 230, 230, 255);
	border-style: outset;
    border-width: 2px;
    border-color: rgba(234, 234, 234, 255);
 }
</string>
  </property>
  <widget class="QPushButton" name="pushButton_02_buyitnow">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>430</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Buy it now</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_01_bid">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>430</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Bid</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_00_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>580</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;Name&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_02_buyoutprice_head">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Buyout Price (THB):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_03_bidprice_head">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bid Price (THB):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_05_description_head">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>130</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Discription:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_05_description">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>155</y>
     <width>280</width>
     <height>265</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_bidprice">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>430</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_02_buyoutprice">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>50</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;buyoutprice&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_03_bidprice">
   <property name="geometry">
    <rect>
     <x>639</x>
     <y>80</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;bidprice&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_04_seller">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>110</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;seller_username&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_04_seller_head">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Seller:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_00_timeleft">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0d 0hr 0m 0s</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget_qquickview" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>55</y>
     <width>460</width>
     <height>410</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
    </font>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>textEdit_05_description</tabstop>
  <tabstop>pushButton_02_buyitnow</tabstop>
  <tabstop>pushButton_01_bid</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
